examples/hpc-enterprise-slurm.yaml

```diff
@@ -78,6 +108,10 @@ deployment_groups:
       instance_image:
         family: $(vars.family)
         project: $(vars.project)
+      service_account:
```

Comment:
IIUC this PR is stating an opinion that service account email should not be bundled with scopes. Can we create a task to update API throughout HPC Toolkit to reflect this opinion.

Reply:
Done